Frontend/Gmfctn/src/app/modules/dashboard/dashboard.component.ts

```diff
@@ -13,12 +13,10 @@ export class DashboardComponent implements OnInit {
 
   ngOnInit(): void {
     this.calculateSize();
-}
+  }
 
   calculateSize(): void {
-
     this.height = ( document.body.clientHeight - 200 ) / 7;
     this.margin = this.height / 7 + '';
```

Comment:
avoid using "[magic numbers](https://codeburst.io/software-anti-patterns-magic-numbers-7bc484f40544)"
it is not understandable what 200 and 7 are. create two const valus with descriptive name and assign these values.

it will be something like that (its only example, provide namings that describe your values):
```
private calculateSize(): void {
    const headerHeight = 200;
    const rowsCount = 7;

    this.height = (document.body.clientHeight - headerHeight) / rowsCount;
    this.margin = (this.height / rowsCount).toString; // i believe that + '' is for conversion to string
}
```

Reply:
Fixed.